src/js/components/Button/StyledButtonKind.js

```diff
@@ -215,6 +216,7 @@ const plainStyle = (props) => css`
     > svg {
       display: flex;
       align-self: center;
+  vertical-align: middle;
```

Comment:
small thing but this should be indented to align with `align-self`

Reply:
Oops, I will go back and fix that